data/css/news.scss

```diff
@@ -62,19 +62,6 @@ $support-font: 'Fira Sans' !default;
     background-color: $background-dark-color;
 }
 
-.ArrangementSideBySide {
-    .CardTitle {
-        &:hover .CardTitle__title {
```

Comment:
Looks like my previous comment about hover should go here! We should port this to wherever should go now.